eclipse-collections/src/main/java/org/eclipse/collections/impl/bimap/mutable/SynchronizedBiMap.java

```diff
@@ -269,18 +267,6 @@ public V forcePut(K key, V value)
         }
     }
 
-    @Override
-    public RichIterable<K> keysView()
-    {
-        return LazyIterate.adapt(this.keySet());
-    }
-
-    @Override
-    public RichIterable<V> valuesView()
-    {
-        return LazyIterate.adapt(this.values());
-    }
-
```

Comment:
It's odd that the implementation of keysView, valuesView, and keyValuesView got split up in some cases. These were moved up to the superclass and all the implementations are together now.